### ISSUE TITLE
Copy location of newly-generated file to clipboard

### ISSUE THREAD

**Issue by jkymarsh** [jakemmarsh/jenerate#6]
After generating a new primary file (React component, Angular controller, etc.), it might be kinda dope to automatically copy the location of the new file to the clipboard, so it could be immediately pasted into a `script` tag or `require` statement or whatever.

**Comment by jakemmarsh** [comment]
https://github.com/xavi-/node-copy-paste

**Comment by jakemmarsh** [comment]
Since our current configuration structure allows for any number of files to be generated for a module (main file, test file, etc), how would we decide which file to copy the path of? First defined?

**Comment by jkymarsh** [comment]
Yeah, or user-defined as part of the config? Boolean flag on a "shouldCopyToClipboard" property or some shit?
